src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -134,11 +134,6 @@ internal sealed partial class TexlBinding
         /// </summary>
         public const int MaxSelectsToInclude = 100;
 
-        /// <summary>
-        /// Default name used to access a Lambda scope.
-        /// </summary>
-        internal static DName ThisRecordDefaultName => new DName("ThisRecord");
```

Comment:
>ThisRecordDefaultName [](http://example.com/codeflow?start=30&length=21)

Would be interesting to know if this breaks PAClient builds? 
That's ok  the build can update. But it may mean extra churn. 

Rather than move this to a new definition on FunctionScopeInfo,  consider avoiding new one and just using existing one.